lib/landrush/cap/linux/add_iptables_rule.rb

```diff
@@ -3,7 +3,7 @@ module Cap
     module Linux
       module AddIptablesRule
         def self.add_iptables_rule(machine, rule)
-          _run(machine, %Q(iptables -C #{rule} 2> /dev/null || iptables -A #{rule}))
+          _run(machine, %Q(/sbin/iptables -C #{rule} || /sbin/iptables -A #{rule}))
```

Comment:
The default path on CentOS v5 is "/usr/kerberos/bin:/usr/local/bin:/bin:/usr/bin".